controller/linodemachine_controller_helpers.go

```diff
@@ -109,6 +109,17 @@ func (r *LinodeMachineReconciler) newCreateConfig(ctx context.Context, machineSc
 		createConfig.Interfaces = slices.Insert(createConfig.Interfaces, 0, *iface)
 	}
 
+	if machineScope.LinodeMachine.Spec.PlacementGroupRef != nil {
```

Comment:
I think we may want to have the PlacementGroupRef be on the LinodeCluster instead like with VPCRef.

Reply:
so we only want the PG on the control-plane nodes, easier to do it on the machine isn't it?

Reply:
It's not about it being easier, it's that it will be a common reference between multiple machines in the cluster like the VPCRef is

Reply:
with the current limitations you cannot put an entire cluster in a placementGroup unless it is very small, it feels more suited to individual machines or groups of machines such as in a machineDeployment

Reply:
We only use MachineDeployment for the workers, looks like we'll want to keep it on LinodeMachine so we can use it on LinodeMachineTemplate for the control-plane

Reply:
yeah, and we also can't update the spec for MachineDeployment